### ISSUE TITLE
Cannot replicate the Key Point metrics.

### ISSUE THREAD

**Issue by harinandan1995** [nileshkulkarni/csm#17]
Hi,
As part of a college project, we implemented CSM using the latest packages and renderers.
While we were able to generate visually good looking UV mappings the Percentage Correct Key Points metric values for the categories bird and car are only at 10% for the alpha 0.1.
We were wondering if there were any training tricks used by the author to get the number up to the 50s.
Also is it possible to provide the trained model dictionary at least for the bird or car category?. So that we can test if its the model that's causing the low PCK or something else.

Thanks

**Comment by nileshkulkarni** [comment]
Hi, 
 The cachedir [here](https://github.com/nileshkulkarni/csm/blob/master/csm/docs/training.md) contains the pre-trained models. You can use the instructions in [testing.md](https://github.com/nileshkulkarni/csm/blob/master/csm/docs/testing.md) to evaluate the model.

Best,
Nilesh

**Comment by harinandan1995** [comment]
Hi @nileshkulkarni,
Thanks for the reply. 
While I was able to load the pre-trained model and run the testing, I am still not able to get the results published in the paper. I would like to know if these pre-trained models are the ones used to get the results published in the paper or if these are some intermediate models. 

Thanks

**Comment by nileshkulkarni** [comment]
Hi @harinandan1995,

Are you using the testing script as part of the repo that is provided to replicate the keypoint metrics? We have provided the models that replicate the results in the paper as it is.

Best,
Nilesh
